packages/ui/react-ui-editor/src/hooks/yjs/yjs-cursor-converter.ts

```diff
@@ -0,0 +1,21 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import * as Y from 'yjs';
+
+import { arrayToString, stringToArray } from '@dxos/util';
+
+import { type CursorConverter } from '../../components';
+
+export const yjsCursorConverter = (text: Y.Text): CursorConverter => ({
```

Comment:
are we intending to completely remove support for YJS? (i.e., this is temp and therefore not it the extensions folder like automerge)?